### ISSUE TITLE
[Search][Suggestions] DuckDuckGo gives an unexpected response to certain search queries

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/android-components#618]
mozilla-mobile/focus-android#3127

It looks like DuckDuckGo will give an empty response to certain search queries. We have to account for this in the Parser

<img width="1326" alt="image" src="https://user-images.githubusercontent.com/1250545/44162978-cbc3ec00-a076-11e8-80d7-876d687e3523.png">

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-5156)

**Comment by boek** [comment]
So I ended up just catching the error in this case. Ultimately I would like to take that out so that I don't swallow actual errors so we can be more responsive when things potentially change and break. Though, in the future I'm not sure how well just an error will help us track down an issue as in this case we got steps to reproduce.

Definitely up for suggestions (ha ha) on a better way to handle this 😄
